utils/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { type ClassValue, clsx } from 'clsx';
+import { twMerge } from 'tailwind-merge';
+
+export function cn(...inputs: Array<ClassValue>): string {
+  return twMerge(clsx(inputs));
+}
+
+export function floorNumber(number: number, numberOfDecimals: number): number {
```

Comment:
`floorNumber` sounds like it would do the same as `Math.floor`, perhaps this should be called `floorToDecimalPlaces`? Also, handy tip: whenever you have a function like this that accepts multiple parameters of the same type, it can be easy to get them mixed up, but if you take in an object with named keys it can be a lot more expressive:
```ts
floorNumberToDecimalPlaces({ number: 5.4321, numberOfDecimals: 2 })
```